undertow/src/main/java/org/wildfly/extension/undertow/ListenerResourceDefinition.java

```diff
@@ -154,8 +154,8 @@ abstract class ListenerResourceDefinition extends PersistentResourceDefinition {
     static final OptionAttributeDefinition RECEIVE_BUFFER = OptionAttributeDefinition.builder("receive-buffer", Options.RECEIVE_BUFFER).setAllowExpression(true).setValidator(new IntRangeValidator(1)).build();
     static final OptionAttributeDefinition SEND_BUFFER = OptionAttributeDefinition.builder("send-buffer", Options.SEND_BUFFER).setAllowExpression(true).setValidator(new IntRangeValidator(1)).build();
     static final OptionAttributeDefinition KEEP_ALIVE = OptionAttributeDefinition.builder("tcp-keep-alive", Options.KEEP_ALIVE).setAllowExpression(true).build();
-    static final OptionAttributeDefinition READ_TIMEOUT = OptionAttributeDefinition.builder("read-timeout", Options.READ_TIMEOUT).setDefaultValue(new ModelNode(60000)).setAllowExpression(true).setMeasurementUnit(MeasurementUnit.MILLISECONDS).build();
-    static final OptionAttributeDefinition WRITE_TIMEOUT = OptionAttributeDefinition.builder("write-timeout", Options.WRITE_TIMEOUT).setDefaultValue(new ModelNode(60000)).setAllowExpression(true).setMeasurementUnit(MeasurementUnit.MILLISECONDS).build();
+    static final OptionAttributeDefinition READ_TIMEOUT = OptionAttributeDefinition.builder("read-timeout", Options.READ_TIMEOUT).setDefaultValue(new ModelNode(1800000)).setAllowExpression(true).setMeasurementUnit(MeasurementUnit.MILLISECONDS).build();
```

Comment:
this is 1800000 millis = 1800 sec, not 180 sec.

Reply:
ooh - good catch @chengfang

Reply:
Fixed @chengfang - great catch!